### PR TITLE
ENH: Add CI.

### LINKED ISSUE #23: Add CI
Add Continuous Integration to the module.

### PR DESCRIPTION
**Pull request by jhlegarreta**
- Add `*.yml` configuration files for CI.
- Add Python wrapping files.
- Update the `README.rst` file to reflect the CI status and Python
  installation steps.
- Modify the current `CTestConfig.cmake` file to make the build results be
  sent to the **Insight** project in **open.cdash.org**.

Resolves #23.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Also pinging @phcerdan since he did lots of wrapping for his ITKIsotropicWavelets module. Thanks.

**Comment by jhlegarreta** [comment]
Not sure whether classes templated only over the dimensionality have the correct wrapping, or else, other wrapping schemes like the one in [itkLabelGeometryImageFilter.wrap](https://github.com/InsightSoftwareConsortium/ITK/blob/c95b99852a9109a8fc176a06f88b2aa6d442154b/Modules/Nonunit/Review/wrapping/itkLabelGeometryImageFilter.wrap) should be used 😔

File: itk-module.cmake [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
-   "${DOCUMENTATION}"
+  DEPENDS
+    ITKImageFeature
+    ITKLevelSets
+    ITKStatistics
+    ITKIOSpatialObjects
+    ITKRegionGrowing
+    ITKLabelVoting
+    ITKMathematicalMorphology
+    ITKIOBruker
```
Comment (dzenanz) [inline itk-module.cmake]: These should probably be replaced by `ITKImageIO`, as mentioned [here](https://discourse.itk.org/t/find-all-available-modules-from-groupio/351/12?u=dzenanz).

File: itk-module.cmake [hunk 2]
```diff
@@ -1,32 +1,38 @@
...
+    ITKIOMeta
+    ITKIOMINC
+    ITKIOGE
+    ITKIOBioRad
+    ITKIOHDF5
+    ITKIOLSM
+    ITKIOMRC
+    ITKIOStimulate
+    ITKVtkGlue
+    ITKTestKernel #to handle IO in src
```
Comment (dzenanz) [inline itk-module.cmake]: And `ITKTestKernel` should be removed.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    print('Please run:', file=sys.stderr)
+    print('', file=sys.stderr)
+    print('  python -m pip install scikit-build')
+    sys.exit(1)
+
+setup(
+    name='lesionsizingtoolkit',
+    version='0.0.1',
+    author='Xiao Xiao Liu',
+    author_email='xiaoxiao.liu@kitware.com',
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply setup.py]: Thanks for the note Matt. Changed in #28. If @XiaoxiaoLiu is willing to provide us with a valid email address, I'd would be willing to change it. For the time being, we can merge the PR though.

**Pull request merged**